genprof/main.cpp

```diff
@@ -183,6 +183,7 @@ bool write_job_profile(string basename, string workspace_base)
                     "  /opt/android-{ndk,sdk}/** rix,\n"
                     "  /usr/share/sbt-launcher-packaging/** rix,\n"
                     "  /usr/sbin/pbuilder pix,\n"
+                    "  /usr/share/meson/** rx,\n"
```

Comment:
Fixed in https://github.com/tux3/jenkins-autoarmor/commit/8fb73a615c42c1a08dc14fa46fa774256b2f3d60